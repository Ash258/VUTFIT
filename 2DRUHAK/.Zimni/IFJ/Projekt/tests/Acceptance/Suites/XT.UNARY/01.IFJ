' Assignment with operators

scope
	dim a as double
	a += 2.0e0
	print a;
	a *= 4.00e0
	print a;
	a -= 4
	print a;
	a /= 2
	print a;
	dim b as integer = 48
	b \= 12
	print b;

	dim c as string = !"bar"
	c += !" foo" + !" "
	print c;
end scope
