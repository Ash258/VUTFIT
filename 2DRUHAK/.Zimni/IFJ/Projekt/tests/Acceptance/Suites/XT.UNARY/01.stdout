 2 8 4 2 4bar foo 